client/components/select-dropdown/test/index.js

```diff
@@ -1,253 +1,215 @@
-			expect( dropdown.instance().focused ).toEqual( 1 );
-		} );
-
-		test( 'permits to select an option by pressing ENTER, or SPACE', () => {
-			function runNavigateItemTest( keyCode ) {
-				const dropdown = shallowRenderDropdown();
-				const activateItemSpy = jest.spyOn( dropdown.instance(), 'activateItem' );
-				const keyEvent = createKeyEvent( keyCode );
-
-				dropdown.find( '.select-dropdown__container' ).simulate( 'keydown', keyEvent );
-				expect( dropdown.state( 'isOpen' ) ).toEqual( true );
-				expect( activateItemSpy ).toBeCalledTimes( 1 );
-			}
-
-			const enterKeyCode = 13;
-			const spaceKeyCode = 32;
-
-			[ enterKeyCode, spaceKeyCode ].forEach( runNavigateItemTest );
-		} );
-
-		test( 'permits to close the dropdown by pressing ESCAPE', () => {
-			const escapeKeyCode = 27;
-			const escEvent = createKeyEvent( escapeKeyCode );
-
-			const dropdown = mountDropdown( true );
-			dropdown.setState( { isOpen: true } );
-
-			const container = dropdown.find( '.select-dropdown__container' );
-			container.simulate( 'keydown', escEvent );
-			expect( dropdown.state( 'isOpen' ) ).toEqual( false );
-			// check that container was focused
-			expect( container.instance() ).toEqual( document.activeElement );
-			dropdown.unmount();
-		} );
-
-		describe( "permits to open the dropdown, and navigate through the dropdown's options by", () => {
-			function runNavigateItemTest( { keyCode, nextFocused } ) {
-				const keyEvent = createKeyEvent( keyCode );
-				const dropdown = mountDropdown();
-				dropdown.instance().focused = 1;
-
-				dropdown.find( '.select-dropdown__container' ).simulate( 'keydown', keyEvent );
-				expect( dropdown.state( 'isOpen' ) ).toEqual( true );
-
-				dropdown.find( '.select-dropdown__container' ).simulate( 'keydown', keyEvent );
-				expect( dropdown.instance().focused ).toEqual( nextFocused );
-			}
-
-			const arrowUp = { keyCode: 38, nextFocused: 0 };
-			const arrowDown = { keyCode: 40, nextFocused: 2 };
-
-			[ arrowUp, arrowDown ].forEach( runNavigateItemTest );
-		} );
-	} );
-
-	/**
-	 * Utilities
-	 */
-
-	function mountDropdown( attach = false ) {
-		const dropdownOptions = getDropdownOptions();
-		return mount(
-			<SelectDropdown options={ dropdownOptions } />,
-			attach ? { attachTo: document.body } : undefined
-		);
-	}
-
-	function shallowRenderDropdown( props ) {
-		const dropdownOptions = getDropdownOptions();
-		return shallow( <SelectDropdown options={ dropdownOptions } { ...props } /> );
-	}
-
-	function getDropdownOptions() {
-		return [
-			{ value: 'status-options', label: 'Statuses', isLabel: true },
-			{ value: 'published', label: 'Published' },
-			{ value: 'scheduled', label: 'Scheduled' },
-			{ value: 'drafts', label: 'Drafts' },
-			null,
-			{ value: 'trashed', label: 'Trashed' },
-		];
-	}
-
-	function createKeyEvent( keyCode ) {
-		return {
-			keyCode,
-			preventDefault: jest.fn(),
-		};
-	}
+function getDropdownOptions() {
```

Comment:
Not a biggie, but I don't see a good reason for this to be a function - we could just declare its return value to be a constant, and use that constant everywhere instead of invoking the function. It doesn't matter for a test, obviously, but theoretically it yields the nice benefit of not declaring the array on every invocation.

Reply:
Agreed. It's basically a fixture which can be declared as a constant. Done via https://github.com/Automattic/wp-calypso/pull/63966/commits/2840ddf6f8ba4be84ac7b207d012cfc61d68c4ed